documentation/development.md

```diff
@@ -0,0 +1,59 @@
+# Git Town Development
+
+## Requirements
+
+* Ruby 2.1.5
+  (install [directly](https://www.ruby-lang.org/en/documentation/installation),
+  or via a ruby manager like [rvm](https://rvm.io/)
+  or [rbenv](https://github.com/sstephenson/rbenv))
+* [ShellCheck](https://github.com/koalaman/shellcheck)
+
+
+## Setup
+
+* fork and clone repository to your machine
+* `bundle` to install ruby gems
+
+
+## Running Tests
+
+* Tests are written in [Cucumber](http://cukes.info/) and [RSpec](http://rspec.info/).
+* We have linters for bash (source), ruby (tests), and cucumber (tests)
```

Comment:
Ideas:

* make "bash" a link to https://github.com/Originate/git-town/blob/master/bin/lint
* make "ruby" a link to https://github.com/Originate/git-town/blob/master/.rubocop.yml
* make "cucumber" a link to your Gem